Gemfile.lock

```diff
@@ -146,7 +146,7 @@ GEM
       activesupport (>= 3.0.0)
     faraday (0.12.2)
       multipart-post (>= 1.2, < 3)
-    ffi (1.9.18)
+    ffi (1.9.21)
```

Comment:
@h-lame, @sihugh I'm seeing [this error][1] when `bundle install` attempts to build native extensions for this new version of `ffi` on my development VM. I've tried getting the latest `govuk-puppet` and running `govuk_puppet` on my development VM, but it didn't fix the problem. I found [this issue][2] which looks like it's the same problem and I ran the command suggested in [this comment][3] to fix the problem: `sudo apt install libffi-dev`. However, I wonder whether we'll see the same problem when we try to deploy this change to staging/production - I wonder whether the AWS integration environment somehow has this package already installed...?

[1]: https://gist.github.com/floehopper/3bff4424a7d033321b8d1ba245e55160
[2]: https://github.com/ffi/ffi/issues/608
[3]: https://github.com/ffi/ffi/issues/608#issuecomment-364168097

Reply:
@floehopper this should be fixed in https://github.com/alphagov/govuk-puppet/pull/7276

Reply:
@sihugh Ah, I hadn't seen that, thanks. Does that mean that Whitehall won't be deployable until that change is applied?